db/migrate/20221111141747_create_utterances.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUtterances < ActiveRecord::Migration[7.0]
+  def change
+    create_table :utterances do |t|
+      t.text :utterance_text
```

Comment:
@DeboDevelop - Can we rename it to `content`?

Reply:
@AdityaTiwari2102 Done.